### DISCUSSION TITLE
Using a config outside of the project being analysed

### DISCUSSION THREAD

**Discussion by geoff-maddock** [golangci/golangci-lint-action#649] [Q&A]
I'm looking to add this linter workflow to a number of go projects.  
I'd rather not have to add the same config to each project, but instead manage the config from one place.  
I have my workflow yaml in a separate repo where it's called by the workflow for each project - is there a way for me to pass in a config file that exists in the context of the golangci-linter action?

**Comment by bombsimon** [comment]
There's no easy way for this currently. Have a look at https://github.com/golangci/golangci-lint/pull/2136 and https://github.com/golangci/golangci-lint/issues/1452 which are some of the discussions on this topic. 

[Here's](https://github.com/golangci/golangci-lint/issues/2137#issuecomment-1050609151) a workaround until a potential native support exist.
